lib/cogctl/actions/relays/delete.ex

```diff
@@ -14,20 +14,28 @@ defmodule Cogctl.Actions.Relays.Delete do
   end
 
   defp do_delete(endpoint, relay_names) do
-    Enum.reduce(relay_names, 0, fn(relay_name, acc) ->
+    results = Enum.reduce(relay_names, %{success: [], failure: []}, fn(relay_name, acc) ->
```

Comment:
Yeah, it would. Looking further, there's some other stuff that'd need to change, too. Never mind about this for now; let's get the code merged.